test/build_wincrypt_test.c

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
If I understand things correctly, this (alongside the [include/openssl/types.h](https://github.com/openssl/openssl/pull/17172/files#diff-b9ce3dfb2da51f37daed01ef8b189286ca436b87b7331cf156f5dace16a7b515) change) is not so much about HPKE.  I would make that into a separate PR, and chances are that it may go in much more quickly, as it seems like a simple bug fix that only needs a small adjustment.

Reply:
as above.